wordpress-mobile/WordPress-Android
# Issue/glide fallback placeholder

#7975

Fixes missing placeholder and error images in the Reader. Behavior is basically copied from `WPNetworkImageView.showDefaultImage()` and `WPNetworkImageView.showErrorImage()`.

To test:
Placeholder
1. Open Reader
2. Scroll down and observe grey rectangle is displayed when an image is being loaded (you might need to adjust network connection quality in the emulator)

Error
1. Turn on air plane mode
2. Open Reader - make sure images are not cached
3. Notice grey rectangle is displayed instead of an empty ImageView.
